tests/test_admin.py

```diff
@@ -842,6 +843,73 @@ def test_menuitem_changelist_redirects_if_not_latest_version(self, mocked_messag
         self.assertRedirectsToVersionList(response, version.content)
         self.assertDjangoErrorMessage("Version is not a draft", mocked_messages)
 
+    def test_menuitem_changelist_messages_endpoint_exists(self):
+        """
+        The messages endpoint provides async code with access to the django message queue.
+        Endpoint is required to fetch update messages after common treebeard operations
+        (like moving a tree node) without reloading page.
+        """
+        menu_content = factories.MenuContentWithVersionFactory()
+        messages_endpoint = reverse(
+            "admin:djangocms_navigation_menuitem_message_storage", args=(menu_content.id,)
+        )
+
+        response = self.client.get(messages_endpoint)
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
We should also check that the endpoint returns something, we've had issues in the past where the endpoint returns a 200 but the result is wrong.

Reply:
No problem, I can easily put in a dummy entry to add a message to django messages queue. Will mock up an update to a navigation treebeard node to test it.